src/components/compliance/Compliance.js

```diff
@@ -990,165 +812,203 @@ const Artefacts = ({
+    }
   </Box>
 }
 Artefacts.displayName = 'Artefacts'
 Artefacts.propTypes = {
-  components: PropTypes.arrayOf(PropTypes.object).isRequired,
-  ocmNodes: PropTypes.arrayOf(PropTypes.object).isRequired,
+  aggregatedOcmNodes: PropTypes.arrayOf(PropTypes.object).isRequired,
+  setAggregatedOcmNodes: PropTypes.func.isRequired,
+  selectedAggregatedOcmNodes: PropTypes.arrayOf(PropTypes.object).isRequired,
+  setSelectedAggregatedOcmNodes: PropTypes.func.isRequired,
+  type: PropTypes.string.isRequired,
+  setType: PropTypes.func.isRequired,
+  ocmRepo: PropTypes.string,
   addOrUpdateFilter: PropTypes.func.isRequired,
   removeFilter: PropTypes.func.isRequired,
-  selectedOcmNodes: PropTypes.arrayOf(PropTypes.object).isRequired,
-  setSelectedOcmNodes: PropTypes.func.isRequired,
-  ocmRepo: PropTypes.string,
   filterMode: PropTypes.string.isRequired,
   toggleFilterMode: PropTypes.func.isRequired,
-  setPersonalFilters: PropTypes.func.isRequired,
-  setMountRescoring: PropTypes.func.isRequired,
   mountRescoring: PropTypes.bool.isRequired,
-  personalFilters: PropTypes.arrayOf(PropTypes.object).isRequired,
-  type: PropTypes.string,
-  setType: PropTypes.func,
+  setMountRescoring: PropTypes.func.isRequired,
+  refreshComplianceSummary: PropTypes.func.isRequired,
 }
 
 
 const ComplianceTab = ({
   component,
   ocmRepo,
 }) => {
-  const [dependencies, state] = useFetchBom({
+  const token = JSON.parse(localStorage.getItem(TOKEN_KEY))
+
+  const [complianceSummary, complianceSummaryState, refreshComplianceSummary] = useFetchComplianceSummary({
     componentName: component.name,
     componentVersion: component.version,
     ocmRepo: ocmRepo,
-    populate: fetchBomPopulate.ALL,
   })
 
   const [type, setType] = React.useState(artefactMetadataTypes.VULNERABILITY)
   const [filterMode, setFilterMode] = React.useState(filterModes.CUSTOM)
 
-  const [personalFilters, setPersonalFilters] = React.useState([]) // init "lazy" as artefact-metadata must be present
+  const personalFilters = [
+    {
+      id: 'filter-responsibility',
+      filter: (aggregatedOcmNode) => {
+        // If responsibles are still loading (i.e. undefined), evaluate to true to pessimistically
+        // determine responsibility. If responsibles have been loaded, those falsy declared artefacts
+        // will be filtered out.
+        if (aggregatedOcmNode.responsibles === undefined) return true
+
+        return aggregatedOcmNode.responsibles.some((responsible) => {
+          return responsible.githubUsers.some((githubUser) => (
+            githubUser.username.toLowerCase() === token.sub.toLowerCase()
+            && githubUser.host === token.github_oAuth.host
+          )) || responsible.emails.some((email) => (
+            email.toLowerCase() == token.github_oAuth.email_address.toLowerCase()
+          ))
+        })
+      }
+    },
+    {
+      id: 'filter-severity',
+      filter: (aggregatedOcmNode) => filterForSeverity({
+        severity: SEVERITIES.MEDIUM,
```

Comment:
should we hardcode severity?

Reply:
Generally, I agree this should not be hardcoded. However, this is equivalent to the currently hardcoded filter which severities are displayed in the BoM tab (-> `LOW` is filtered-out, only `MEDIUM` and higher are displayed). And _therefore_ we already have a separate request to allow configuration of the minimum severity, so I'd suggest I'll add a TODO here and we'll come back to it later.